components/Common/Banner/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo } from 'react';
+import { sanitize } from 'isomorphic-dompurify';
+import { dateIsBetween } from '../../../util/dateIsBetween';
+import { isAbsoluteUrl } from '../../../util/isAbsoluteUrl';
+import styles from './index.module.scss';
+import { WebsiteBanner } from '../../../types';
+
+export interface BannerProps {
+  bannersIndex: WebsiteBanner;
+}
+
+const useTextContent = ({ text, link, bannerBtnText }: WebsiteBanner) =>
+  useMemo(() => {
+    if (text) {
```

Comment:
Might be another disconnect with the Storybook vs the actual usage, but these banner displays are based off the start/end dates in the JSON that dictate how long the banner is shown for a campaign

Reply:
@nschonni That part is actually being checked here https://github.com/nodejs/nodejs.org/pull/5233/files#diff-ce3ccfab991d0d5b74fb505155f7ae195213df71c5ccd974fef7e7153a2e5681R52